docs/source/routing/self-hosted/containerization/aws.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Deploying GraphOS Router on Amazon Web Services
+subtitle: Deploy router with Amazon Elastic Container Service (ECS) 
+description: Build and deploy Apollo Router on Amazon Web Services (AWS) with Amazon Elastic Container Service (ECS)
+---
+
+Learn how to deploy the router for development on AWS with Elastic Container Service (ECS).
+
+You will:
+- Build a router image with a Dockerfile.
+- Set up an Elastic Cloud Registry and push your router image to it.
+- Create an ECS task definition for your router and deploy it.
+
+<Note>
+
+Not deploying to AWS? Try our router deployment guides for [GCP](/graphos/routing/self-hosted/containerization/gcp) or [Azure](/graphos/routing/self-hosted/containerization/azure).
+
+</Note>
+
+## Prerequisites
+
+Before you start:
+
+1. [Set up a GraphQL API in GraphOS](/graphos/get-started/guides/graphql#step-1-set-up-your-graphql-api).
+    - Save your `APOLLO_KEY` and `APOLLO_GRAPH_REF`. You'll need them when deploying the router.
+1. Install [Docker](https://www.docker.com/get-started/) locally.
+1. Set up you [AWS environment](https://aws.amazon.com/getting-started/guides/setup-environment/)
+    - Install the AWS CLI.
+    - Use an existing Create an Amazon 
```

Comment:
I think this is what you were going for:

```suggestion
    - Use an existing account or create one.
```

Separately--do we want to provide links for these or just allow users to find the right place to sign up/install the CLI?